lib/mobility-core/src/Kernel/External/SMS/Interface/ExotelSms.hs

```diff
@@ -51,7 +51,9 @@ sendOTP exoCfg SendSMSReq {..} = do
         BasicAuthData
           (DT.encodeUtf8 apiKey)
           (DT.encodeUtf8 apiToken)
-  res <- Ex.sendOTPApi exoUrl authData sid exoOtpSmsTemplate exoPhoneNumber senderName
+      exoSmsType = toUrlPiece IT.Transactional
```

Comment:
why to call `toUrlPiece` here?

Reply:
for converting it into desired text format

Reply:
But why to even use this? It should be automatically converted from instance?